<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>authentication</class>
 <widget class="QDialog" name="authentication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Start game</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icon.ico</normaloff>:/images/icon.ico</iconset>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:  url(:/images/gamefon.png);
background-repeat: no-repeat;
text-transform: uppercase; 
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_game">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>181</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/arrow-right.png);
background-repeat: no-repeat;
border: 0;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="plr1_rchange">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>30</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;
background-image: url(:/images/arrow-r.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="player1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>71</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/player1.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="plr1_lchange">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>410</y>
     <width>30</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;
background-image:url(:/images/arrow-l.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="player2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>350</y>
     <width>71</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/player2.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="plr2_lchange">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>410</y>
     <width>30</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;
background-image:url(:/images/arrow-l.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="plr2_rchange">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>410</y>
     <width>30</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;
background-image: url(:/images/arrow-r.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>510</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
border: 2px solid #fff; 
color: #fff;
padding-left: 30px;</string>
   </property>
   <property name="text">
    <string>Player1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>510</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
border: 2px solid #fff; 
color: #fff;
padding-left: 30px;</string>
   </property>
   <property name="text">
    <string>Player2</string>
   </property>
  </widget>
  <widget class="QLabel" name="ball">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>290</y>
     <width>81</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/images/ball.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="ball_rchange">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>340</y>
     <width>30</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;
background-image: url(:/images/arrow-r.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ball_lchange">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>30</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;
background-image:url(:/images/arrow-l.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
